content/en/docs/guides/mozilla-sops.md

```diff
@@ -188,6 +188,62 @@ Note that you shouldn't apply the encrypted secrets onto the cluster with kubect
 
 Encrypting with age follows the same workflow than PGP. See the [Kustomization-controller API doc](/docs/components/kustomize/kustomization/#age) for more details.
 
+
+## Encrypting secrets using HashiCorp Vault
+
+Sops needs a valid Vault token in an environment variable named `VAULT_TOKEN` to decrypt the secrets.
```

Comment:
We should make kustomize-controller load the token from the secret and feed it to our [SOPS decryptor](https://github.com/fluxcd/kustomize-controller/tree/main/internal/sops).

Reply:
created an issue: https://github.com/fluxcd/kustomize-controller/issues/516

Reply:
thanks